src/styles/app.styl

```diff
@@ -507,9 +507,60 @@ footer
       font-weight bold
       color $blue
 
+      &[aria-busy=true]
+          position relative
+
+          &::after
+              position static
+              width      (16/base-font-size)em
+              height     (16/base-font-size)em
+              margin-left (8/base-font-size)em
+              margin-right (-24/base-font-size)em
+              @extend    $icon-spinner-blue
+
 .service-logo--maif
     box-sizing border-box
     width 4.25rem
     height 4.25rem
     padding .5rem .5rem .75rem
     background-color $maif
+
+
+// Modal
+.modal-overlay
+  position         fixed
+  top              0
+  left             0
+  width            100vw
+  height           100vh
+  z-index          20
+  overflow         hidden
+  background-color rgba(0,0,0,.5)
+  display          flex
+  align-items      center
+  justify-content  center
+  transition       background-color .3s
+
+  .modal
+    margin-top    0px
+    max-width     800px
+    width         80%
+    height        80%
+    background    white
+    border-radius 1em
+    box-shadow    0 1px 3px .7 black
+    transition    margin-top .3s
+
+  &[aria-hidden=true]
```

Comment:
The modal life time is very short, it's inserted in DOM and removed after being hidden. So it's easier to make the transition effect starting directly with an overlay with transparent background.
Using `display: none` would cause to first change the `display` property, and then the `background-color` to trigger the transition.